spec/param.json

```diff
@@ -28,6 +28,10 @@
             "description": "An optional boolean signifying the parameter should be obscured when displayed.",
             "type": "boolean",
             "default": false
-        }
+        },
+        "param": [
```

Comment:
I'm a tad confused, shouldn't this be an array with [ rather than {?

Since we're passing:
```yaml
    param:
      - /spec/containers/0/param
```

similar to yaml ^^?